src/insns/wavedrom/csr-instr.adoc

```diff
@@ -5,7 +5,7 @@
 ....
 {reg: [
   {bits: 7,  name: 'opcode',   attr: ['7', 'SYSTEM=1110011'], type: 8},
-  {bits: 5,  name: 'rd',       attr: ['5', 'dest'], type: 2},
+  {bits: 5,  name: 'rd/cd',    attr: ['5', 'dest'], type: 2},
   {bits: 3,  name: 'funct3',   attr: ['3', 'CSRRS=010', 'CSRRC=011', 'CSRRWI=101', 'CSRRSI=110', 'CSRRCI=111'], type: 8},
   {bits: 5,  name: 'rs1/uimm', attr: ['5', 'source', 'source', 'uimm[4:0]', 'uimm[4:0]', 'uimm[4:0]'], type: 4},
```

Comment:
Shouldn't this be cs1 as well for cap mode?

Reply:
Agreed, I think rs1 continues to be rs1. @arichardson : Can we resolve this and merge the fix?